Gemfile

```diff
@@ -174,10 +174,9 @@ end
 
 group :test do
   gem 'rspec-html-matchers'
-  gem 'rspec-rails', require: false
   # Rails performance tests
```

Comment:
The default cucumber matchers were resulting in very unhelpful error messages, so imported the rspec ones, which are much more useful.